.github/workflows/cxx-build.yml

```diff
@@ -36,9 +44,12 @@ jobs:
 
   openrpc:
     name: Build Openrpc Artifacts
+    needs: thunder
     runs-on: ubuntu-latest
     container:
-      image: kevinshahfws/node-c:3.0
+      image: node:latest
```

Comment:
Can we manually specify whatever the latest version is here? It wouldn't be the first time that "latest" changed to something that broke a pipeline.

Reply:
fixed